plugin-api/src/main/java/io/everytrade/server/plugin/api/parser/ImportedTransactionBean.java

```diff
@@ -17,19 +17,22 @@
     private final Currency quote;
     private final TransactionType action;
     private final Instant imported = Instant.now();
+    private final String note;
 
-    public ImportedTransactionBean(
+    protected ImportedTransactionBean(
```

Comment:
Why protected?

Reply:
Why the change was needed?

Reply:
I is an abstract class and only subclasses can call super constructor.